sru/release-27.7/test_world_readable_logs.sh

```diff
@@ -0,0 +1,58 @@
+series=$1
+deb=$2
+
+set -e
```

Comment:
For bash to capture the error inside a function like what would happen if one of the `find ` commands fails, I think we need this here
```suggestion
set -eE
```